homework11/main.js

```diff
@@ -0,0 +1,145 @@
+// Bài 1
+// Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+//Cách 1
```

Comment:
```suggestion
// Cách 1
```
ko được dùng tiếng Việt